MekHQ/src/mekhq/AtBGameThread.java

```diff
@@ -318,7 +318,8 @@ public void run() {
 
                     // we need to wait until the game has actually started to do transport loading
                     // This will load the bot's infantry into APCs
-                    Thread.sleep(MekHQ.getMekHQOptions().getStartGameDelay());
+                    // sometimes it needs more time?
+                    Thread.sleep(MekHQ.getMekHQOptions().getStartGameDelay() * 2);
```

Comment:
This makes no sense... the option exists so you can set the time based on your setup, and if it is having communication troubles you should just increase the option's value

Reply:
Maybe I'm being excessively cynical, but I imagine most people don't touch the default setting for that, and have no idea what it means. Thus, the sequence of events I'm picturing is: 1. Person sees units not loading into transport. 2. Person goes onto github/discord/slack to complain about it. 3. Somebody has to waste their time explaining about the delay setting, maybe successfully maybe not. I'd like to skip parts 1-3 of this process.

Another option is to get rid of this, but set the default delay to 2x or even 3-4x higher. I'm currently leaning towards that.

A third option is to go back to sending entities over one at a time, which will most likely guarantee correct behavior, but will be considerably slower in the general case.

Reply:
You probably aren't being cynical but instead realistic... I'd go with 4x the default as a base if this is as much of an issue as it sounds.

Reply:
Narrowed down the actual cause of the problem, so now we basically wait for all the bot's units to load before attempting to load transports.